### PR TITLE
Integration list page and integration config

### PR DESCRIPTION
**Pull request by gaspergrom**
# Changes proposed ✍️

### What
copilot:summary
​
copilot:poem

### How
copilot:walkthrough

## Checklist ✅
- [ ] Label appropriately with `Feature`, `Improvement`, or `Bug`.
- [ ] Add screenshots to the PR description for relevant FE changes
- [ ] New backend functionality has been unit-tested.
- [ ] API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).
- [ ] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.

### CODE REVIEW

File: frontend/src/config/integrations/discord/components/discord-connect.vue [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+const connectUrl = computed(() => config.discordInstallationUrl);
+
+const connect = () => {
+  window.open(connectUrl.value, '_self');
+};
+
+const finallizeDiscordConnect = () => {
+  const {
+    source, code,
+  } = route.query;
```
Comment (joanagmaia) [inline frontend/src/config/integrations/discord/components/discord-connect.vue:34]: I don't think route is defined anywhere

File: frontend/src/modules/admin/modules/integration/config/status/connecting.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { IntegrationStatusConfig } from '@/modules/admin/modules/integration/config/status/index';
+
+const connecting: IntegrationStatusConfig = {
+  key: 'connecting',
+  show: (integration: any) => integration.status !== undefined,
```
Comment (joanagmaia) [inline frontend/src/modules/admin/modules/integration/config/status/connecting.ts:5]: Is this ok?
Currently we are showing both integrations done + in progress. Shouldn't we simply include the ones in progress?

File: frontend/src/config/integrations/discord/components/discord-connect.vue (hunk 1 above)
Comment (gaspergrom) [reply frontend/src/config/integrations/discord/components/discord-connect.vue:34]: it is you can catch route.query from anywhere and get the params

File: frontend/src/modules/admin/modules/integration/config/status/connecting.ts (hunk 2 above)
Comment (gaspergrom) [reply frontend/src/modules/admin/modules/integration/config/status/connecting.ts]: fixed

**Review by joanagmaia — APPROVED** [review]

**Pull request merged**